src/stratigraphy/util/predictions.py

```diff
@@ -274,6 +286,43 @@ def evaluate(self, ground_truth_layers: list):
                 layer.material_is_correct = False
                 layer.depth_interval_is_correct = None
 
+    def evaluate_metadata(self, metadata_ground_truth: dict):
+        """Evaluate the metadata of the file against the ground truth.
+
+        Note: For now coordinates is the only metadata extracted and evaluated for.
+
+        Args:
+            metadata_ground_truth (dict): The ground truth for the file.
+        """
+        if self.metadata.coordinates is None:
+            self.metadata_is_correct["coordinates"] = False
+        elif metadata_ground_truth is None or metadata_ground_truth.get("coordinates") is None:
+            self.metadata_is_correct["coordinates"] = None
+
+        else:
+            if (
+                self.metadata.coordinates.east.coordinate_value > 2e6
+                and metadata_ground_truth["coordinates"]["E"] < 2e6
+            ):
+                ground_truth_east = int(metadata_ground_truth["coordinates"]["E"]) + 2e6
+                ground_truth_west = int(metadata_ground_truth["coordinates"]["N"]) + 1e6
+            elif (
+                self.metadata.coordinates.east.coordinate_value < 2e6
+                and metadata_ground_truth["coordinates"]["E"] > 2e6
+            ):
+                ground_truth_east = int(metadata_ground_truth["coordinates"]["E"]) - 2e6
+                ground_truth_west = int(metadata_ground_truth["coordinates"]["N"]) - 1e6
+            else:
+                ground_truth_east = int(metadata_ground_truth["coordinates"]["E"])
+                ground_truth_west = int(metadata_ground_truth["coordinates"]["N"])
+
+            if (int(self.metadata.coordinates.east.coordinate_value) == ground_truth_east) and (
+                int(self.metadata.coordinates.north.coordinate_value) == ground_truth_west
+            ):
```

Comment:
Do we get better results if we allow the absolute value of the difference to be <= 2? Like this, we might catch some cases where conversions between LV03 and LV95 were already applied in the ground truth.

Reply:
Deal with this in a follow up ticket.